content/guides/pgpool.md

```diff
@@ -106,7 +106,7 @@ It's also possible to specify the lifetime of a Pgpool-II child process (`CC_PGP
 
 Use Pgpool-II to distribute the load and separate **WRITE** and **READ** queries between your PostgreSQL servers.
 
-When using the **Streaming** mode, it's not Pgpool-II that manages the replication of your databases. Replication must be set up on the PostgreSQL side, otherwise known as **binary**, **Hot Standby** or **Streaming** replication. To schedule this procedure, you can contact our support at <support@clever-cloud.com> or make a request via our [sales form](https://www.clever-cloud.com/en/contact-sales).
+When using the **Streaming** mode, it's not Pgpool-II that manages the replication of your databases. Replication must be set up on the PostgreSQL side, otherwise known as **binary**, **Hot Standby** or **Streaming** replication. To schedule this procedure, you can contact [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice) or make a request via our [sales form](https://www.clever-cloud.com/en/contact-sales).
```

Comment:
Wasn't sure of who must do the set up, tried to specify it here.

```suggestion
When using the **Streaming** mode, it's not Pgpool-II that manages the replication of your databases. Clever Cloud engineers must manually set up the replication on the PostgreSQL side, otherwise known as **binary**, **Hot Standby** or **Streaming** replication. To schedule this procedure, you can contact [Clever Cloud Support](https://console.clever-cloud.com/ticket-center-choice) or make a request via our [sales form](https://www.clever-cloud.com/en/contact-sales).
```